src/components/Article.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ title, date, text }) => (
+  <article>
+    <h2>
+      {`${title}`}
```

Comment:
No need for extra string conversion.
```suggestion
      {title}
```